linkerd/app/inbound/src/http/router.rs

```diff
@@ -11,34 +11,34 @@ use linkerd_app_core::{
 use std::{borrow::Borrow, net::SocketAddr};
 use tracing::{debug, debug_span};
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct Http {
+    port: u16,
+    settings: http::client::Settings,
+    tls: tls::ConditionalServerTls,
+}
+
 #[derive(Clone, Debug)]
-struct RequestTarget {
+struct LogicalPerRequest {
     addr: Remote<ServerAddr>,
     tls: tls::ConditionalServerTls,
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
-struct Target {
+struct Logical {
     logical: Option<NameAddr>,
     addr: Remote<ServerAddr>,
     http: http::Version,
     tls: tls::ConditionalServerTls,
 }
 
 #[derive(Clone, Debug)]
-struct Logical {
+struct Profile {
```

Comment:
new names are much better :+1:

i might want to see comments saying "this is used as a target for the xxx stack", for first-time readers, but not a blocker.